frontend/src/services/eventHandlers/gameHandler.js

```diff
@@ -76,6 +77,13 @@ export default class GameHandler {
                 );
                 EventBus.$emit(Event.CORRECT_GUESS, player);
             }
+        } else if (payload.gameMasterAPI === "scoreTime") {
+            store.commit("scribble/setRoundNumber", payload.scoreTime.round);
+        } else if (payload.gameMasterAPI === "requestCurrentGameInfo") {
+            store.commit(
+                "scribble/setRoundNumber",
+                payload.requestCurrentGameInfo.round
```

Comment:
If this is gonna get implemented, we should update as much as possible here. i.e. try to use all our data available.